src/main/java/com/example/jwt/service/UserServiceImp.java

```diff
@@ -1,8 +1,19 @@
 package com.example.jwt.service;
 
 import com.example.jwt.entity.User;
+import com.example.jwt.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
 
-public interface UserServiceImp {
+@Component
+@RequiredArgsConstructor
+public class UserServiceImp implements UserServiceImp {
```

Comment:
you forgot it, it should implement UserService not it's own self.

Reply:
Sorry, I fucked that part